docs/api/base-window.md

```diff
@@ -356,6 +356,8 @@ as `-webkit-app-region: drag` in a frameless window.
 
 Calling `event.preventDefault()` will prevent the menu from being displayed.
 
+To convert `point` to DIP, use [`screen.dipToScreenPoint(point)`](./screen.md#screenscreentodippointpoint-windows).
```

Comment:
If you're converting *to* a DIP, wouldn't you want to use `screen.screenToDipPoint()`?

Also FWIW `screen.screenToDipPoint(point)` and `screen.dipToScreenPoint(point)` seem to currently be marked as Windows only (but I think there's another PR pending for that?)